src/shared/Sidebar/component/DirectMessageList.js

```diff
@@ -0,0 +1,16 @@
+import DirectMessageItem from './DirectMessageItem';
+import './DirectMessageList.scoped.css'
+
+function DirectMessageList ({ showCloseIcon, directMessageList }) {
+  
+    return (
+        <ul>
+            { directMessageList.map(item => {
+                return  <DirectMessageItem item={item.uid} showCloseIcon={showCloseIcon} />
+            }) }

```

Comment:
pwede mo to gawin
`{ directMessageList.map(item => <DirectMessageItem item={item.uid} showCloseIcon={showCloseIcon} />) }`